docs/src/main/sphinx/installation/deployment.rst

```diff
@@ -34,9 +34,9 @@ Linux operating system
 Java runtime environment
 ^^^^^^^^^^^^^^^^^^^^^^^^
 
-Trino requires a 64-bit version of Java 11, with a minimum required version of 11.0.15.
-Earlier patch versions such as 11.0.2 do not work, nor will earlier major versions such as Java 8.
-Newer major versions such as Java 12 or 13, including Java 17, are not supported -- they may work, but are not tested.
+Trino requires a 64-bit version of Java 17, with a minimum required version of 17.0.0.
```

Comment:
I think we should require 17.0.3 since there are various crash bugs fixed and we always test against the latest. If people are going to install a new JDK, it seems best to have them start with the most recent version.

Reply:
👍